### ISSUE TITLE
Upgrading from Alpha 0.2b to 0.2c looses learned recipes

### ISSUE THREAD

**Issue by Hurons** [sinkillerj/ProjectE#360]
I have Alpha 0.2b installed on a SMP server and it works great.
But when I replaced 0.2b with 0.2c we no longer knew any recipes. (Tablet recipe list is now empty)
If this is intentional please close this issue, but that would mean I will never upgrade since I cannot sell this 'feature' to the other players.

Many thanks.

Hurons.

**Comment by sinkillerj** [comment]
Does the issue occur updating to the dev version instead?

**Comment by Hurons** [comment]
Hey SinKillerJ, 

No I have not tried the dev versions yet. But to be safe (and not waste more of your time) I did another test. Deleted all files from my test server, uploaded copy of the live server and started it (with 0.2b).

Learned some recipes.

Stopped server and replaced with 0.2c, started server. 

And I know the same recipes now.

So looks like it's a PICNIC problem on my side :/

Closing issue.
